external_ml/seviri/nn_driver.txt

```diff
@@ -0,0 +1,30 @@
+#PATHS TO CPH/COT MODEL/SCALING FILES
+COT_SCALING_FILEPATH   =  /perm/ms/de/sf7/esa_cci_c_proc/orac_repository/orac_devel/orac/external_ml/seviri/data/SCALER_CMA_14_150_150_1_LOSS-MSE_OPT-ADAM_LR-0001_NE-300_BS-200_GSICS_THEANO.pkl
+
+CPH_SCALING_FILEPATH   =  /perm/ms/de/sf7/esa_cci_c_proc/orac_repository/orac_devel/orac/external_ml/seviri/data/SCALER_CPH_14_40_40_1_LOSS-MSE_OPT-ADAM_LR-0001_NE-300_BS-200_GSICS_THEANO.pkl
+
+COT_MODEL_FILEPATH     =  /perm/ms/de/sf7/esa_cci_c_proc/orac_repository/orac_devel/orac/external_ml/seviri/data/MODEL_CMA_14_150_150_1_LOSS-MSE_OPT-ADAM_LR-0001_NE-300_BS-200_GSICS_THEANO__1.0.4__.h5
+
+CPH_MODEL_FILEPATH     =  /perm/ms/de/sf7/esa_cci_c_proc/orac_repository/orac_devel/orac/external_ml/seviri/data/MODEL_CPH_14_40_40_1_LOSS-MSE_OPT-ADAM_LR-0001_NE-300_BS-200_GSICS_THEANO__1.0.4__.h5
+
+#ML BACKEND TO BE USED (THEANO/TENSORFLOW)
+BACKEND                =  THEANO
+
+# ONLY CHANGE IF YOU KNOW WHAT YOU ARE DOING
+COT_THRESHOLD          = 0.2
```

Comment:
As the network for CMA was trained to predict COT directly the threshold should be a real COT threshold. A bit confusing might be that the outputted COT is in the range [0, 1] even though COTs in reality can be > 1. This is because I limited the CALIOP training target COT also in the range [0, 1] by setting all COTs > 1 to 1. This decision is based on the fact that for COTs > 1 we have a cloud anyways and do not gain any further information. 

Prediciting with a network trained against binary CALIOP CPH should also directly yield a CPH regression value which is then classified to binary by thresholding.

Reply:
having a threshold of cloud/aerosol of 1 COT seems a bit conservative, a lot of saharan dust will be caught up in that threshold  , is this what you notice?  Ideally the variable should have something to indicate its origin in the NN code as I think later on on the L3 code we have some other possibilities of COT thresholds but it is a minor quible.

Reply:
Inside CALIOP files there is a COT and a AOT, so theoretically the CALIOP COT product should not be polluted with aerosol. So limiting the training target COT to 1 should do the job as the binary classification threshold (`COT_THRESHOLD`) will be < 1. The  `COT_THRESHOLD=0.2` is mathematically retrived but I also expected it to be about 0.2 because passive instruments are missing very thin clouds.   

You are right, I will introduce a better name to avoid confusion later on (e.g. `NN_COT_THRESHOLD`, `NN_CPH_THRESHOLD`)

Reply:
Ahh yes that makes sense, agreed that COT threshold for passive intruments is approx 0.2